server/manager/src/main/java/org/apache/accumulo/manager/tableOps/availability/SetTabletAvailability.java

```diff
@@ -97,7 +97,7 @@ public long isReady(FateId fateId, Manager manager) throws Exception {
       }
     };
 
-    var start = NanoTime.now();
+    var start = Timer.startNew();
```

Comment:
later in the class the `start.elapsed().toMillis()` is called.  Could use the time unit version of the method.